<launch>

<node pkg="rosbag" type="play" name="player" args="-l $(find reggie_support)/bags/realsense.bag"/>

<node name="rviz" pkg="rviz" type="rviz" args="-d $(find reggie_support)/rviz/crop_cloud.rviz" />

<node name="crop_cloud_node" pkg="reggie_localize" type="crop_cloud" output="screen" >

  <!-- DEFAULT MAP
  <param name="x_min" type="double" value="-0.726849" />
  <param name="x_max" type="double" value="0.706064" />
  <param name="y_min" type="double" value="-0.543809" />
  <param name="y_max" type="double" value="0.267617" />
  <param name="z_min" type="double" value="1.014" />
  <param name="z_max" type="double" value="1.331" />
  -->

  <!-- YELLOW MAP
  <param name="x_min" type="double" value="0.36" />
  <param name="x_max" type="double" value="0.375" /> 
  <param name="y_min" type="double" value="0.32" />
  <param name="y_max" type="double" value="0.34" />
  <param name="z_min" type="double" value="-10" />
  <param name="z_max" type="double" value="10" />
  -->

  <!-- GREEN MAP
  -->
  <param name="x_min" type="double" value="-0.43" />
  <param name="x_max" type="double" value="-0.42" /> 
  <param name="y_min" type="double" value="0.35" />
  <param name="y_max" type="double" value="0.365" />
  <param name="z_min" type="double" value="-10" />
  <param name="z_max" type="double" value="10" />

  <!-- BLUE MAP
  <param name="x_min" type="double" value="0.34" />
  <param name="x_max" type="double" value="0.35" /> 
  <param name="y_min" type="double" value="-0.29" />
  <param name="y_max" type="double" value="-0.28" />
  <param name="z_min" type="double" value="-10" />
  <param name="z_max" type="double" value="10" />
  --> 
  <param name="camera_topic" type="str" value="/camera/depth/color/points" />
  <param name="camera_frame" type="str" value="camera_depth_optical_frame" />
</node>

</launch>
